tests/integration/test_summary.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import sqlite3
+from contextlib import closing
+
+import pytest
+
+from airbase.fetch import fetch_json
+from airbase.resources import E1A_SUMMARY_URL
+from airbase.summary.db import summary
+
+
+@pytest.fixture(scope="module")
+def db_dump() -> list[dict[str, str]]:
+    with sqlite3.connect(summary()) as db, closing(db.cursor()) as cur:
+        cur.execute(
+            """
+            SELECT
+                country_code,
+                pollutant,
+                cast(pollutant_id AS TEXT)
+            FROM
+                summary;
+            """
+        )
+        return [dict(ct=ct, pl=pl, shortpl=shortpl) for ct, pl, shortpl in cur]
+
+
+def test_summary(db_dump: list[dict[str, str]]):
+    assert db_dump == fetch_json(E1A_SUMMARY_URL)
```

Comment:
I love this approach, very clean and simple ✨

Reply:
I like to put the test setup on a fixture even if it is a one off,
as it makes the setup/test separation very clear.